WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -130,39 +129,33 @@ public async Task<bool> StartRoundAsync(IEnumerable<SmartCoin> smartCoins, Round
 			return finalRoundState.WasTransactionBroadcast;
 		}
 
-		private AliceClient CreateAliceClient(SmartCoin coin, RoundState roundState)
+		private async Task<ImmutableArray<AliceClient>> CreateRegisterAndConfirmCoinsAsync(IEnumerable<SmartCoin> smartCoins, RoundState roundState, CancellationToken cancellationToken)
```

Comment:
i'm confused, doesn't this this need to be `ImmutableArray<AliceClient?>`?

Reply:
We are returning the not null ones 
https://github.com/molnard/WalletWasabi/blob/bbbb23c5d7790813c8cebabf1743d95ad4a52299/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L162

Reply:
lol, how did i miss that